libs/robotis-cm-300/device.d.ts

```diff
@@ -0,0 +1,28 @@
+declare namespace pins {
+    //% fixedInstance shim=pxt::getPin(PIN_LED1)
```

Comment:
PIN_LED ?

Reply:
That's because our board designer named it LED1.
However, I modified the LED to look better.

Done on 271bc7e.